src/data/reducers/cohorts.test.js

```diff
@@ -51,10 +47,9 @@ describe('cohorts reducer', () => {
       errorFetching: false,
       finishedFetching: true,
     };
-    expect(cohorts(undefined, {
-      type: GOT_COHORTS,
-      cohorts: cohortsData,
-    })).toEqual(expected);
+    expect(
+      cohorts(undefined, actions.fetching.received(cohortsData)),
```

Comment:
I know that cohortsData is copied over, but I feel it is dangerous to have in here, as it can be out-dated without breaking the tests (or the reducer), while sitting there as a flawed example of the data model that people base assumptions on.
Recommend an array of arbitrary objects to test against: { a: 'cohort', another: 'COHort' }, etc